contrib/docker/docker-compose.prod.yaml

```diff
@@ -2,6 +2,11 @@ version: "3.6"
 services:
   reverse-proxy:
     image: traefik:v2.8
+    depends_on:
+      play:
+        condition: service_healthy
+      back:
+        condition: service_healthy
```

Comment:
There is no reason to wait for play and back services to be up to start Traefik!

Traefik adapts to containers when they start and adds those automatically.
If you do so, if, for instance, back is down, play won't be reachable.